src_app/catalog_app.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { Linking } from 'react-native';
```

Comment:
What happens here with Lumin?

Reply:
Probably nothing because a platform have to handle deep-link, in this class we see only usage of it. Anyway, I cannot check it to be sure.

Reply:
I think we cannot/shouldn't use ```import from 'react-native'``` in a shared code (like catalog_app.js).